esp/src/src-react/components/Menu.tsx

```diff
@@ -206,6 +233,8 @@ export const SubNavigation: React.FunctionComponent<SubNavigationProps> = ({
     const [isFavorite, addFavorite, removeFavorite] = useFavorite(window.location.hash);
     const [history] = useHistory();
 
+    const [nextPrev] = useNextPrev();
```

Comment:
I'd checked this branch out locally, but I'm not seeing the next / previous buttons anywhere. It seems like this nextPrev from the hook is always defined as the serialization of an object ("[object Object]")